server/grpc_service.go

```diff
@@ -895,6 +895,7 @@ func (s *GrpcServer) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error
 			continue
 		}
 
+		transportDuration := time.Now().Unix() - int64(request.GetInterval().GetEndTimestamp())
```

Comment:
Do not relay on this, it's may be different system time in the host.

Reply:
yes, the latency metrics already exits , I will remove it .